client/state/current-user/gravatar-status/test/reducer.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	GRAVATAR_UPLOAD_REQUEST,
+	GRAVATAR_UPLOAD_REQUEST_SUCCESS,
+	GRAVATAR_UPLOAD_REQUEST_FAILURE,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+import reducer, {
+	isUploading
+} from '../reducer';
+
+describe( 'reducer', () => {
+	it( 'exports expected reducer keys', () => {
+		expect( reducer( undefined, {} ) ).to.have.keys( [
+			'isUploading',
+		] );
+	} );
+
+	describe( '#isUploading', () => {
+		it( 'returns false by default', () => {
+			const state = isUploading( undefined, {} );
+			expect( state ).to.equal( false );
+		} );
+
+		it( 'returns true when request is made', () => {
```

Comment:
We can probably trim this down a bit since we know previous state doesn't change this reducer. eg: the default state, plus one each for each action type.